assets/js/pluginsidebar/services/media/getImageThumbnail.spec.js

```diff
@@ -0,0 +1,28 @@
+/* global describe, expect, it */
+
+import getImageThumbnail from './getImageThumbnail';
+
+describe('getImageThumbnail', () => {
```

Comment:
Need to ensure that Jest is installed as a devDependency and that it's being run in Travis.